lightning/backend/local.go

```diff
@@ -1510,3 +1498,181 @@ func (s *sizeProperties) iter(f func(p *rangeProperty) bool) {
 		return f(prop)
 	})
 }
+
+type LocalWriter struct {
+	db                *pebble.DB
+	writeErr          common.OnceError
+	local             *LocalFile
+	lastKey           []byte
+	produceWg         sync.WaitGroup
+	consumeWg         sync.WaitGroup
+	kvsChan           chan []common.KvPair
+	sstDir            string
+	memtableSizeLimit int64
+}
+
+func (w *LocalWriter) isSorted(kvs []common.KvPair) bool {
+	if len(kvs) <= 1 {
+		return false
```

Comment:
Because if it is only one, maybe the keys come soon will not be in order.

Reply:
There are two scenarios that can lead to only one key in a batch:
- The kv is quite big, exceed the batch send threshold(default to 64k)
- it is the last key in the chunk

In either case, I think it's ok to still do the sorted check to avoid adding one-key wb.

Or， maybe we should delay the sst writers phase until the sorted kv paired are large enough to be flushed, thus we can always do check for all keys

Reply:
Ok, I will check it.

Reply:
The region concurrency will be large. It is not a good idea to keep too much keys in memory...